payjoin/src/core/receive/mod.rs

```diff
@@ -205,7 +209,26 @@ impl InputPair {
             return Err(InternalPsbtInputError::InvalidScriptPubKey(AddressType::P2tr).into());
         }
 
-        Self::new_segwit_input_pair(txout, outpoint, sequence, None)
+        let expected_weight = InputWeightPrediction::P2TR_KEY_DEFAULT_SIGHASH.weight()
+            + Weight::from_non_witness_data_size(32 + 4 + 4);
```

Comment:
This is defined as a const in several places https://github.com/search?q=repo%3Apayjoin%2Frust-payjoin%20Weight%3A%3Afrom_non_witness_data_size(32%20%2B%204%20%2B%204)&type=code
Can we de-duplicate this somehow?